README.md

```diff
@@ -1,9 +1,25 @@
-# ARCHILATOR
+# 🅰🆁🅲🅷🅸🅻🅰🆃🅾🆁
 
-A simple terminal-based calculator
+A simple terminal-based calculator 🧮
 
-### Dependencies
+## Dependencies
+  - [node.js](nodejs.org)
+  - [npm](https://www.npmjs.com/)
 
-### Setup
+## Setup  
+1. Download the .zip file or clone the repo using: 
+      
+        gh repo clone chris-chen-creates/archilator
+2. Install the dependencies using:
+   
+        npm i
 
-### How to run
\ No newline at end of file
+## How to run
+1. Open a terminal window
+2. Navigate to the Archilator project
+3. Within terminal, use the command:
+   
+        npm start 
+4. Type in some math problems using any of the following characters: `0-9 + - * / . ( ) ^`
```

Comment:
nit: `Type in some math problems` -> `Enter any arithmetic equation into the prompt`

Reply:
changed